### ISSUE TITLE
Notifications from other openhab instances

### ISSUE THREAD

**Issue by RayBa82** [openhab/openhab-cloud#201]
Hi, i dont know if this is the right place for this issue.

I am getting Android notifications from openhab cloud instances which i dont run.
In all languages. 

This is a little bit scaring and I assume there is something running completely wrong.

**Comment by digitaldan** [comment]
Hi this is the same as #184 .  We have an issue where in some corner case the push notification ID is duplicated across different user devices.  This is only a bug in notifications, there is no remote access or other account dangers here.  Can you send me the email you use to log into the service to support@myopenhab.org ?  I will go ahead and remove your entry from the database.  Once this is done the next time you open the mobile app is will register for a new push notification ID.  Thanks.

**Comment by digitaldan** [comment]
Duplicate of #184

**Comment by mkarnek** [comment]
Hi. I just wanted to let you know that I today received a notification from someone else's OpenHab on my iOS device. First and only time it has happened to me since I started to use the service. Thanks.
